configs/nextflow.ngp.config

```diff
@@ -43,6 +52,7 @@ profiles {
 		params.species = "staphylococcus aureus"
 		params.speciesDir = 'saureus'
 		params.mlstScheme = 'saureus'
+		params.symlinkDir = "/access/jasen/saureus/"
```

Comment:
Does it not work if `params.symlinkDir` is not provided?

Reply:
Yea it was a 2am solution to suppress the warnings. Same w the devmode flag. Good catch